Sources/StreamChat/Controllers/DatabaseObserver/StateLayerDatabaseObserver.swift

```diff
@@ -52,19 +52,19 @@ final class StateLayerDatabaseObserver<ResultType: DatabaseObserverType, Item, D
 
 @available(iOS 13.0, *)
 extension StateLayerDatabaseObserver where ResultType == EntityResult {
-    func item() -> Item? {
+    var item: Item? {
         var item: Item?
         context.performAndWait {
             item = Self.makeEntity(frc: frc, context: context, itemCreator: itemCreator, sorting: sorting)
         }
         return item
     }
     
-    func startObserving(initial: Bool = false, didChange: @escaping (Item?) async -> Void) throws {
```

Comment:
`initial` was not useful since `didChange` is async and therefore on object creation it causes a delay. Instead, I am accessing the database observer directly for initial values.